staging/src/k8s.io/metrics/hack/update-codegen.sh

```diff
@@ -28,13 +28,13 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-ge
 #
 # we skip informers and listers for metrics, because we don't quite support the requisite operations yet
 # we skip generating the internal clientset as it's not really needed
-${CODEGEN_PKG}/generate-internal-groups.sh deepcopy,conversion \
+"${CODEGEN_PKG}"/generate-internal-groups.sh deepcopy,conversion \
   k8s.io/metrics/pkg/client k8s.io/metrics/pkg/apis k8s.io/metrics/pkg/apis \
   "metrics:v1alpha1,v1beta1 custom_metrics:v1beta1 external_metrics:v1beta1" \
-  --output-base "$(dirname ${BASH_SOURCE})/../../.." \
-  --go-header-file ${SCRIPT_ROOT}/hack/boilerplate.go.txt
-${CODEGEN_PKG}/generate-groups.sh client \
+  --output-base "$(dirname "${BASH_SOURCE[0]}")/../../.." \
+  --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate.go.txt
+"${CODEGEN_PKG}"/generate-groups.sh client \
```

Comment:
please quote the whole string for these

Reply:
Thank you for your reminding. I have modified it. Thank you